crates/swc/tests/fixture/shopify/004-jest/output/index.js

```diff
@@ -8,11 +8,11 @@ Object.defineProperty(exports, "App", {
         return App;
     }
 });
-var _interopRequireDefault = require("@swc/helpers/lib/_interop_require_default.js").default;
-var _slicedToArray = require("@swc/helpers/lib/_sliced_to_array.js").default;
-var _react = /*#__PURE__*/ _interopRequireDefault(require("react"));
-var _reactI18N = require("@shopify/react-i18n");
+var _interop_require_default = require("@swc/helpers/lib/_interop_require_default.js").default;
+var _sliced_to_array = require("@swc/helpers/lib/_sliced_to_array.js").default;
+var _react = /*#__PURE__*/ _interop_require_default(require("react"));
+var _react_i_1_8n = require("@shopify/react-i18n");
```

Comment:
Is this intended?

Reply:
Yes, the optional choice here is using snake_case only for swc helpers or using snake_case for all.
I have opted for the latter.
Which option do you prefer?

Reply:
I find the `_react_i_1_8n` is really strange.

Reply:
Yeah, I think the latter is fine, but I think `_i_1_8_n` would disturb debugging

Reply:
It seems like there's no good alternative.
`convert_case` produces `react_i_18_n`